tools/bin/ecflow/processes/main/post_processing/swe_plot.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python
+###### plotting swe percentiles
+###### usage: <python> <swe_plot.py> <configuration.cfg>
+
+###### Creates a SWE percential spatial plot 
+###### Compares one day of VIC output to 30 years of data
+###### Using the Weibull Plotting Position
```

Comment:
this script needs:

1.  PEP8 cleanup
2.  separate analysis and plotting code.
3.  a bit more documentation